scripts/set_up_design_system.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+#
+# Downloads and sets up the Design System components
+#
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "${DIR}"/.. || exit
+
+if [ $# -eq 0 ] || [ "$1" == "" ]; then
+    echo "Usage: set_up_design_system.sh {TAG_NAME}"
+elif [ "$1" == "" ]; then
+    TAG_NAME=$(curl "https://api.github.com/repos/${REPO_NAME}/releases" | jq '.[0].name' | tr -d '"')
+else
+    TAG_NAME="$1"
+fi
+
+REPO_NAME="onsdigital/design-system"
+DOWNLOAD_URL=$(curl "https://api.github.com/repos/${REPO_NAME}/releases/tags/${TAG_NAME}" | jq '.assets[0].browser_download_url' | tr -d '"')
+RELEASE_NAME=${DOWNLOAD_URL##*/}
+
+echo "Fetching ${DOWNLOAD_URL}"
+
+TEMP_DIR=$(mktemp -d)
+
+curl -L --url "https://github.com/${REPO_NAME}/releases/download/${TAG_NAME}/${RELEASE_NAME}" --output "${TEMP_DIR}/${RELEASE_NAME}"
+unzip -o "${TEMP_DIR}/${RELEASE_NAME}" -d .
+rm -rf "${TEMP_DIR}"
+
+#
+# Now load the print stylesheet
+#
+CDN_URL=${CDN_URL:-"https://cdn.ons.gov.uk"}
+PRINT_STYLE_SHEET_FILE_PATH_PATH=${PRINT_STYLE_SHEET_FILE_PATH_PATH:-"cms/jinja2/assets/styles"}
+
+echo "Loading print style sheets from CDN for DS Version ${TAG_NAME}"
+
+PATH_TO_FETCH="${CDN_URL}/sdc/design-system/${TAG_NAME}/css/print.css"
```

Comment:
The variable names in the rest of the script are pretty descriptive. `PATH_TO_FETCH` sort of isn't. Perhaps `PRINT_STYLE_SHEET_CDN_URL`?